### ISSUE TITLE
Any chance we could get semantic version numbering?

### ISSUE THREAD

**Issue by joshgoebel** [rbreaves/kinto#714]
major.minor.patch

1.2.13 vs 1.2-13

Context: Writing a READMe explaining how people can use keyszer right now with Kinto for testing if they wish.

**Comment by rbreaves** [comment]
Sure, I don't see why not. That make a process easier? I guess it is more of the norm. I was thinking of the patch version being like a specific build number in my mind, but it doesn't matter much to me moving it over 1.x.x and shouldn't break any sort of tooling I have.

**Comment by joshgoebel** [comment]
> I guess it is more of the norm.

Yeah, kind of that.  And if you get serious you make promises about when you might break things and when you won't... that part is both liberating and free at the same time.
